### PR TITLE
Add sponsor to JEP table

### PR DESCRIPTION
**Pull request by bitwiseman**
@kohsuke

### CODE REVIEW

**Comment by kohsuke** [comment]
🎉  I think sponsors should be celebrated

File: jep/201/README.adoc [hunk 1]
```diff
@@ -19,15 +19,10 @@ endif::[]
...
-| https://github.com/ewelinawilkosz2[Ewelina Wilkosz] from https://github.com/praqma[Praqma]
+| link:https://github.com/ewelinawilkosz2[Ewelina Wilkosz] from link:https://github.com/praqma[Praqma]
 
 | Status
 | Draft :speech_balloon:
-//| Rejected :no_entry:
-//| Withdrawn :hand:
-//| Final :lock:
-//| Replaced :dagger:
-//| Active :smile:
```
Comment (daniel-beck) [inline jep/201/README.adoc]: Doesn't this break the status script?

File: jep/306/README.adoc [hunk 2]
```diff
@@ -34,12 +34,14 @@ endif::[]
 //
 // Uncomment if there is an associated placeholder JIRA issue.
 | JIRA
-| https://issues.jenkins-ci.org/browse/JENKINS-50294[JENKINS-50294]
+| link:https://issues.jenkins-ci.org/browse/JENKINS-50294[JENKINS-50294]
```
Comment (daniel-beck) [inline jep/306/README.adoc]: Aren't these redundant? On the jenkins.io site, URLs with protocol are automatically links.

File: jep/201/README.adoc (hunk 1 above)
Comment (bitwiseman) [reply jep/201/README.adoc]: Nope.  This isn't needed for the status script anymore.

**Pull request merged**